css/templates/gallery/components/_hero.css

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+.gallery-component-hero {
+  margin-bottom: calc(60 / 16rem);
```

Comment:
all calculations need to be documented well or use variables to demonstrate how we arrived at this value. Otherwise this will cause UI to break even with small changes